galaxy/tools/verify/interactor.py

```diff
@@ -228,7 +228,10 @@ def new_history(self):
 
     @nottest
     def test_data_path(self, tool_id, filename):
-        return self._get("tools/%s/test_data_path?filename=%s" % (tool_id, filename)).json()
+        admin_key = self.master_api_key
+        response = self._get("tools/%s/test_data_path?filename=%s" % (tool_id, filename), key=admin_key)
```

Comment:
I think you should use `admin=True` instead?

Reply:
Yes, indeed. Thanks for the catch.